modules/post/hardware/automotive/can_flood.rb

```diff
@@ -0,0 +1,43 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+class MetasploitModule < Msf::Post
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'can_flood',
+        'Description' => 'Module that floods a CAN interface',
+        'License' => MSF_LICENSE,
+        'Author' => ['Pietro Biondi'],
+        'DisclosureDate' => ['March 20 2019'],
```

Comment:
This should not be an Array. Also, ISO date format is objectively superior.

```suggestion
        'DisclosureDate' => '2019-03-20',
```

That said, the disclosure date usually represents the date that the "vulnerable" functionality was initially implemented or the technique was additionally discovered... but I'm not sure how that would be applicable here.

Reply:
Correct, regarding the date, I could also remove it.

Reply:
`DisclosureDate` is a mandatory field for `exploit` modules. I *think* it can be safely removed from `post `modules.

Reply:
Yes, I confirm, it can be safely removed from ```post``` modules. Thanks